apps/src/blockly/googleBlocklyWrapper.ts

```diff
@@ -706,6 +710,18 @@ function initializeBlocklyWrapper(blocklyInstance: GoogleBlocklyInstance) {
 
     blocklyWrapper.navigationController.addWorkspace(workspace);
 
+    blocklyWrapper.getNewCursor = function (type) {
+      switch (type) {
+        case 'basic':
+          return new blocklyWrapper.BasicCursor() as Cursor;
```

Comment:
do we need to cast all of these cursor types? `blocklyWrapper.Cursor()` should be the right type already, right? Does `BasicCursor` extend from `Cursor`?

Reply:
It looks like it's happy without the casting. `LineCursor` extends `BasicCursor` which extends `Cursor`. Will remove!